adapters/tpmn/tpmn.go

```diff
@@ -0,0 +1,159 @@
+package tpmn
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/prebid/openrtb/v19/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+// TpmnAdapter struct
+type adapter struct {
+	URI string
```

Comment:
`adapter` is private member of `tpmn` package. `URI` can also be private
```
type adapter struct {
	uri string
}
```

Reply:
update commit : 
[458faa1](https://github.com/prebid/prebid-server/pull/2947/commits/458faa1a7c7943115695ce8d227f8abaa283d7fe)